### ISSUE TITLE
Weather clock

### ISSUE THREAD

**Issue by Steph1116** [jake1164/Pico-RGB-Matrix-Weather-Clock#24]
Just updated everything from the old code that just had the date, time, day. I'm using the latest weather src you added. I've added all needed information in the settings.toml file. Added the whole src to my pico w and ran the code. I have the splash screen (sun) then the display goes black. 

Thonny shows correct output of all info supposed to be displayed, however nothing is being displayed. 

I did however see for a brief moment the display did display the weather, time, conditions. But then it went back to a black screen.

**Comment by jake1164** [comment]
Is it dark?  Could it be the autodimming feature that turns the display off at night?
________________________________
From: Steph1116 ***@***.***>
Sent: Sunday, March 12, 2023 11:32 AM
To: jake1164/Pico-RGB-Matrix-Clock ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jake1164/Pico-RGB-Matrix-Clock] Weather clock (Issue #24)


Just updated everything from the old code that just had the date, time, day. I'm using the latest weather src you added. I've added all needed information in the settings.toml file. Added the whole src to my pico w and ran the code. I have the splash screen (sun) then the display goes black.

Thonny shows correct output of all info supposed to be displayed, however nothing is being displayed.

I did however see for a brief moment the display did display the weather, time, conditions. But then it went back to a black screen.

**Comment by Steph1116** [comment]
Nothing is currently on the display, so maybe it's the auto dimming,I'm currently looking to turn that feature off.

**Comment by Steph1116** [comment]
Bingo, now I have the issue where I set to imperial or metric and the temp shows a °F Not showing °C

**Comment by jake1164** [comment]
That has worked pretty well on my two clocks so I would be interesting in knowing how light / dark it is where it is.  It is a setting in the settings page, however, since those settings are not stored it wont keep between reboots.

Did you add the new settings to the TOML file for Open Weather Maps?
________________________________

**Comment by jake1164** [comment]
That could very well be.. we could add some debug to the light sensor to find out where the level is and allow it to be adjusted in the TOML file.
________________________________

**Comment by Steph1116** [comment]
For my daily use I am blacking out line 3, turning line 1 and 2 red and still using it as a night clock.

**Comment by Steph1116** [comment]
To temporarily fix °C I just replaced the letter F with C as I won't ever need F

**Comment by Steph1116** [comment]
![image](https://user-images.githubusercontent.com/122418750/224568682-7597e376-7826-4ed0-98fc-28290863a4e8.jpeg)

I've settled on this for now. Much appreciated all the help and the latest code update. Added hours of fun playing around with this today. I'm going to see how the image icon effects my dark room at night, I may want to look into getting rid of it all together unfortunately.

I changed the layout for my setup as I feel it sits better whereas the scrolling text doesn't clash with the bottom of the pm. 

Looking forward to any now changes you'll add later!

Thanks again!
